src/app/[locale]/components/FormProgress.tsx

```diff
@@ -29,31 +33,26 @@ const FormProgress = () => {
 
   if (step < FormStep.STEP_5_THANK_YOU) {
     return (
-      <div className="relative flex flex-col-reverse sm:flex-col">
-        <div>
-          {step > FormStep.STEP_1_DESCRIPTION && (
-            <Button
-              appearance={'subtle-button'}
-              className="sm:absolute sm:left-0 sm:-top-2 stepper-button-hover pl-0-overwrite"
-              onClick={() => back()}
-            >
-              <FaChevronLeft />
-              {t('back')}
-            </Button>
-          )}
-        </div>
+      <div className="flex flex-col">
+        {step > FormStep.STEP_1_DESCRIPTION && (
+          <Button
+            appearance={'subtle-button'}
+            className="stepper-button-hover pl-0-overwrite"
```

Comment:
Ik vroeg me trouwens nog af hoe ik hier de default hover style het makkelijkst vanaf kan halen + aanpassen?

Reply:
Bedoel je de Tailwind classes eraf halen? Als het om een design keuze zoals kleur gaat, kunnen die er volgens mij sowieso af.

En ik weet niet helemaal zeker of je dit bedoelt maar misschien heb je er wat aan:

Button krijgt nu de (hover) style (bijv. de bg color) vanuit het purmerend thema (`package.json`: `"@nl-design-system-community/purmerend-design-tokens": "1.0.0"` = een `.json` met de Purmerend tokens). Vanuit het Utrecht `<Button>` Component zijn er tokens beschikbaar die ingesteld kunnen worden met de Purmerend huisstijl (tokens in die Purmerend .json).

Er zitten verschillende layers met tokens in die json, even wat screenshotjes uit de inspector die het hopelijk duidelijker (of juist meer complicated.. 🥲 ) maken.

Brand:
<img width="305" alt="Scherm­afbeelding 2024-11-21 om 14 58 29" src="https://github.com/user-attachments/assets/d14eff92-a04f-4d0f-a031-875a5977944e">
Basis:
<img width="508" alt="Scherm­afbeelding 2024-11-21 om 14 58 22" src="https://github.com/user-attachments/assets/55a7d4c2-a4e9-4fca-bfab-549b0578c768">
Common:
<img width="627" alt="Scherm­afbeelding 2024-11-21 om 14 58 09" src="https://github.com/user-attachments/assets/72013f62-7eec-4ef1-9ad9-714825cdeee2">
Component:
<img width="663" alt="Scherm­afbeelding 2024-11-21 om 14 57 52" src="https://github.com/user-attachments/assets/fd7060c0-fd5e-42fb-89be-e7986f686ec4">

En omdat je appearance op de` <Button>` hebt gezet, is dit hoe de token er uiteindelijk uit ziet:
<img width="844" alt="Scherm­afbeelding 2024-11-21 om 14 57 32" src="https://github.com/user-attachments/assets/f877d1d9-dd6e-487c-9557-bca55d49e1a0">

Als je bijv. een andere bg color wil hebben, is de nette optie volgens mij dat je een andere harde waarde uit de brand layer kiest. Ik weet niet zeker of je dat zelf kan doen op dit moment en het zou ook kunnen dat hier een extra token voor nodig is in de Purmerend thema json, not sure. @Marwaxhello is hieraan bezig dus misschien kan zij helpen.

PS: @Robbert please correct me if I'm wrong/niet duidelijk.😌

Reply:
Dit is wel een beetje waar ik op doelde inderdaad, bedankt in ieder geval voor het uitgebreide antwoord. In dit geval gaat het mij meer over de hover state / kleur van de butten in de FormProgress:

Zonder hover:
<img width="593" alt="image" src="https://github.com/user-attachments/assets/daa5caef-55a0-4205-91ef-b213f97d1980">

Met hover:
<img width="685" alt="image" src="https://github.com/user-attachments/assets/00ba4021-9729-4479-bccd-1c8ef1d9ae8e">

Ik vind dit wat overdreven voor een tekst button, maar ik bedenk me dat ik misschien beter gewoon een ander button component hiervoor moet gebruiken. Ik ga zo even kijken of ButtonLink of LinkButton hier meer geschikt voor is

Reply:
Met <LinkButton> gaat het wel goed :)